resources/models.py

```diff
@@ -406,6 +406,7 @@ class Berth(AbstractBoatPlace):
         related_name="berths",
         on_delete=models.PROTECT,
     )
+    comments = models.TextField(verbose_name=_("comments"), blank=True,)
```

Comment:
the name should be `comment`, as field names in plural are used for reverse RoeignKeys and M2M relations 🙂